source/assets/components/RecipeCardMy.js

```diff
@@ -0,0 +1,106 @@
+const $ = (selector) => document.querySelector(selector);
+
+class RecipeCardMy extends HTMLElement {
+  constructor() {
+    super();
+    this.shadow = this.attachShadow({ mode: 'open' });
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group16/cse110-fa21-group16/pullRequest?prid=8393580)
```suggestion
    this.shadow = this.attachShadow({ mode: "open" });
```